portfolio.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="mystyle.css" />
+  </head>
+  <body>
+    <header>
+      <div class="logo"><a href="#Logo">Amanor Djoleto</a></div>
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href="#Portfolio">Portfolio</a></li>
+          <li><a href="#About"> About</a></li>
+          <li><a href="#Contact"> Contact</a></li>
+        </ul>
+      </nav>
+      <img src="./images/Icons/Union.png" alt="Hamburger" class="menu-button" />
+    </header>
+
+    <section class="headline">
+      <div class="info">
+        <h1>Hey there, I am Nii Amanor</h1>
+        <h2>I am a Software Developer</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+    </section>
```

Comment:
- [ ] Good job working on the changes requested by the previous reviewer, While implementing the changes you introduce issues to your design
 - Your background images overflow to the hero section content, Please use an only required background image(the image at the right bottom in the Figma template) and make sure it matches the design on the [Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
 - You removed the social media icons from the design, Please place the social media icons and make sure it matches the design on the [Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). 

### Your design

![image](https://user-images.githubusercontent.com/83803414/158775349-592b6b58-9e54-49a2-816b-83c66068bac4.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/158775297-faf2f3b6-384a-47f2-890e-e97a46b99f22.png)

Reply:
@thecodechaser please can you give advice on how to make the background image work

Reply:
> @thecodechaser please can you give advice on how to make the background image work

@ramrod433 Hello, There are many ways to go but I would suggest You can use `position: absolute` for the image and place it where you want with top, bottom, right, left properties.

Reply:
@thecodechaser thank you very much